src/command.cpp

```diff
@@ -77,11 +77,26 @@ void Command::arg(const char* name, const char* value, int quotes)
     addcomma();
     json.append("\"");
     json.append(name);
-    json.append(quotes ? "\":\"" : "\":");
-    json.append(value);
     if (quotes)
     {
-        json.append("\"");
+        json.append("\":\"");
+        size_t valueLen = strlen(value);
+        for (size_t i = 0; i < valueLen; i++)
```

Comment:
Do we need to take UTF-8 into account here, and increment by character size in bytes?  eg might we alter a character that takes two or more bytes to represent but isn't one of those below?

Reply:
This is a good point. The specs say:
```The representation of strings is similar to conventions used in the C family of programming languages. A string begins and ends with quotation marks. All Unicode characters may be placed within the quotation marks except for the characters that must be escaped: quotation mark, reverse solidus, and the control characters (U+0000 through U+001F).```
Hence, it seems like UTF-8 chars don't need (and should not) be escaped, unless they are the `"`, the `\` or control characters.